app/src/main/java/com/example/market_kurly/feature/home/component/HomeProductRow.kt

```diff
@@ -7,17 +7,16 @@ import androidx.compose.foundation.lazy.LazyRow
 import androidx.compose.foundation.lazy.items
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
-import com.example.market_kurly.feature.home.ProductItem
-import com.example.market_kurly.ui.theme.MARKETKURLYTheme
-
+import androidx.navigation.NavController
+import com.example.market_kurly.domain.model.ProductData
 
 
 @Composable
 fun HomeProductRow (
-    products: List<ProductItem>,
-    modifier: Modifier = Modifier
+    products: List<ProductData>,
+    modifier: Modifier = Modifier,
+    navController: NavController
```

Comment:
```suggestion
    products: List<ProductData>,
    navController: NavController,
    modifier: Modifier = Modifier
```
P1: 인자 순서 이런 식으로 하면 좋겠네요~

Reply:
수정했습니다!